docs/contracts.rst

```diff
@@ -241,7 +241,7 @@ Each Contract Factory exposes the following methods.
         >>> txn_receipt['contractAddress']
         '0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318'
 
-.. py:classmethod:: Contract.constructor(*args, **kwargs).estimateGas(transaction=None, block_identifier=None)
+.. py:classmethod:: Contract.constructor(*args, **kwargs).estimate_gas(transaction=None,block_identifier=None)
```

Comment:
This should still be `estimateGas`. The only estimate gas you changed is the one attached to the eth module. So if the example is `w3.eth.estimate_gas`, that should be updated in the docs, but if it's on contract, like the ones in this file, `estimateGas` is currently correct.

Reply:
Ok, I see. Got it.